src/main/java/it/gov/pagopa/payhub/activities/utility/Utilities.java

```diff
@@ -1,14 +1,24 @@
 package it.gov.pagopa.payhub.activities.utility;
 
+import lombok.Getter;
+
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class Utilities {
-
     private Utilities(){}
     public static final Pattern EMAIL_PATTERN = Pattern.compile("^[a-zA-Z0-9_!#$%&'*+/=?`{|}~^.-]+@[a-zA-Z0-9.-]+$");
     public static final int IBAN_LENGTH = 27;
 
+    @Getter
+    public enum CLASSIFICATION {
+        TES_NO_MATCH("TES_NO_MATCH");
+        final String value;
+        CLASSIFICATION(String value) {
+            this.value = value;
+        }
```

Comment:
if name and value are the same, we could avoid the store the value and use directly the name.
Could you list all the classifications reported on Confluence?
https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/1339031693/Classificazione+incassi

Reply:
done